cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -95,21 +94,19 @@ def get_pr_url(self, base_branch, head_branch):
 
     def fetch_upstream(self):
         """ git fetch <upstream> """
-        self.run_cmd(f"git fetch {self.upstream}")
+        cmd = ["git", "fetch", self.upstream]
+        self.run_cmd(cmd)
 
-    def run_cmd(self, cmd, shell=False):
+    def run_cmd(self, cmd):
         if self.dry_run:
```

Comment:
Just for the case add:
```python
assert not isinstance(cmd, str)
```

Reply:
I think it's a great idea to add an assertion here. Perhaps we should go with positive, instead of the negative? Let me know if you think otherwise and I'll change it.

Reply:
Nevermind, I followed your suggestion in the end because testing that cmd is a sequence but it isn't a string would bloat a simple line with no real benefit.